docs/rules/prefer-spread.md

```diff
@@ -41,6 +41,11 @@ Examples of **correct** code for this rule:
 ```js
 /*eslint prefer-spread: "error"*/
 
+// Using the spread operator
+
```

Comment:
Minor nit: could you get rid of this empty line, as none of the other examples below have one?